chain-impl-mockchain/src/leadership/bft.rs

```diff
@@ -5,7 +5,7 @@ use crate::update::{BftSelectionDiff, ValueDiff};
 use chain_core::property::{self, LeaderSelection};
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
-pub struct BftRoundRobinIndex(usize);
+pub struct BftRoundRobinIndex(pub usize);
```

Comment:
This disclaims any invariants that may be enforced by methods on this type. Sure about exposing the inner integer?

Reply:
To be honest, I don't think this wrapper type is very useful to begin with...

Reply:
I removed the use of BftRoundRobinIndex in GenesisLeaderSelection.